predictoor/README.md

```diff
@@ -0,0 +1,49 @@
+
+Alas, accurate predictions are *hard*. Worse, typical prediction feeds have no accountability on accuracy. If the weatherman says "no rain for today" and then it rains, a farmer could get stuck in the mud, wrecking a portion of his crops. The weatherman doesn't feel the impact of wrong predictions, but the farmers sure care! 
+
+**Accountable Predictions** - Imagine if there was accountability. Accuracy would go up; the farmer would be stuck less. Imagine accountable prediction feeds for not only for rain, but also wind, sea temperature, road congestion, train delays, ETH prices, NVID prices, housing prices, and more. **Imagine tens of thousands of prediction feeds with accountable accuracy.** Imagine them **globally distributed**, and censorship resistant. Imagine **accuracy improving with time**. 
+
+## Introduction
+Ocean Predictoor is a stack and a dapp for prediction feeds. It has accountability for accuracy, via staking. It’s globally distributed and censorship-resistant, by being on-chain. We expect its accuracy to improve over time, due to its incentive structure. Its first use case is DeFi token prediction because users can close the data value-creation loop quickly to make tangible \$.  
+
+Predictoor is composed of different actors, often referred to as: predictoors and traders. You will learn in the [Predictoor Introduction](pdr-intro.md) page, you will out more about these actors, how they are structured in the system, and their behaviors. By the time you've completed reading this, you'll know what Epochs are, and have a better understand of what "t+1" means.
+
+## Architecture
+After obtaining a broad overview of the system and its actors, you should be ready to dive into [The Architecture of Predictoor](pdr-architecture.md) and the [pdr-backend repo](https://github.com/oceanprotocol/pdr-backend). You'll become more exposed to the key building blocks and pdr-backend, the integration component for Predictoors and Traders.
+
+We anticipate Predictoor to extend beyond DeFi to other verticals like climate and agriculture. Predictoor agents may evolve into AI DAOs with emergent swarm-like behavior. This is the future.
+
+## Earning with Predictoor
+Predictoor serves two groups of actors: Predictoors, and Traders. In this section, we'll explore [How to Earn with Predictoor](pdr-earn.md) and describe how both of these groups can benefit from it. After you are done with this, you should know what steps you need to take next in order to start leveraging the system for your own benefit.
+
+We'll also provide some links and exercise to make this easier for you.
+
+## Predictoor Parameters
+The [Predictoor Parameters page](pdr-parameters.md) will provide you with details on many topics such as incentives, feed costs, and deployed feeds.  
+
+You can find details about Oasis, Testnet, Mainnet, and how to bridge your OCEAN here.  
+
```

Comment:
Right here -- after "Predictoor parameters" section and before "Connect with us" -- you should have a paragraph on FAQ. For example:

```
## FAQ

The [FAQ page](pdr-faq.md) has frequently asked questions about Predictoor.
```